azure-iot-device/tests/provisioning/pipeline/test_pipeline_stages_provisioning.py

```diff
@@ -50,6 +50,14 @@ def apply_fake_pipeline_thread(fake_pipeline_thread):
     pass
 
 
+# automatically mock the transport for all tests in this file.
+@pytest.fixture(autouse=True)
```

Comment:
>pytest [](start = 1, length = 6)

was something failing in horton without this ?

Reply:
It turns out this was unnecessary.  I did this as an experiment and never pulled it out.

---
In reply to: [394571019](https://github.com/Azure/azure-iot-sdk-python/pull/499#discussion_r394571019) [](ancestors = 394571019)